packages/table/style.ts

```diff
@@ -24,7 +24,11 @@ export const tableCss = css`
 `;
 
 export const rightGrid = css`
-  background: linear-gradient(to right, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0));
+  background: linear-gradient(
+    to right,
+    ${hexToRgbA(black, 0.2)},
```

Comment:
This works if we don't want to decide on an "official" shadow color yet, but I meant that we should have something like
```
shadow: hexToRgbA(black, 0.2)
```
in `styles/color.ts` and import that here